ui/src/localPeer.ts

```diff
@@ -0,0 +1,61 @@
+import * as remote from "./remote";
+
+// TYPES
+export enum StatusType {
```

Comment:
```suggestion
export enum Status {
```

minor suggestion

Reply:
We have a `Status` type tho.

Reply:
Ah, right I only scrolled down after that suggestion :sweat_smile:  DISREGARD...

Real talk though: these sum types are such boilerplate. Who hurt you, typescript?

Reply:
> Real talk though: these sum types are such boilerplate. Who hurt you, typescript?

It was javascript, never a good idea to be backwards compatible. Which for rescript.

Reply:
Bless this mess :pray: